README.md

```diff
@@ -1,360 +1,81 @@
-
-To pass an option, use:
+Then, create objects on your index:
 
 ```js
-var client = algoliasearch(applicationId, apiKey, {
-  timeout: 4000
-})
-```
-
-### Callback convention
-
-Every API call takes a callback as last parameter. This callback will then be called with two arguments:
-
- 1. **error**: null or an [Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) object. More info on the error can be find in `error.message`.
- 2. **content**: the object containing the answer from the server, it's a JavaScript object
-
-### Promises
-
-**If you do not provide a callback**, you will get a promise (but never both).
-
-Promises are the [native Promise implementation](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Promise).
-
-We use [jakearchibald/es6-promise](https://github.com/stefanpenner/es6-promise) as a polyfill when needed.
-
-### Request strategy
-
-The request strategy used by the JavaScript client includes:
-
-- On the browser:
-  + [CORS](https://en.wikipedia.org/wiki/Cross-Origin_Resource_Sharing#Browser_support) for modern browsers
-  + [XDomainRequest](https://msdn.microsoft.com/en-us/library/ie/cc288060%28v=vs.85%29.aspx) for IE <= 10
-  + [JSONP](https://en.wikipedia.org/wiki/JSONP) in any situation where Ajax requests are unavailable or blocked.
-- Node.js:
-  + native [`http` module](https://nodejs.org/api/)
-
-Connections are always `keep-alive`.
-
-### Cache
-
-**Browser only**
-
-To avoid performing the same API calls twice **search** results will be stored
-in a `cache` that will be tied to your JavaScript `client` and `index` objects.
-Whenever a call for a specific query (and filters) is made, we store the results
-in a local cache. If you ever call the exact same query again, we read the
-results from the cache instead of doing an API call.
-
-This is particularly useful when your users are deleting characters from their
-current query, to avoid useless API calls. Because it is stored as a simple
-JavaScript object in memory, the cache is automatically reset whenever you
-reload the page.
-
-It is never automatically purged, nor can it be completely disabled. Instead, we
-provide the `index.clearCache()` (or `client.clearCache()` if you're using the
-[Search multiple indices](#search-multiple-indices) method that you can call to reset it.
-
-### Proxy support
-
-**Node.js only**
-
-If you are behind a proxy, just set `HTTP_PROXY` or `HTTPS_PROXY` environment variables before starting your Node.js program.
-
-```sh
-HTTP_PROXY=http://someproxy.com:9320 node main.js
+const algoliasearch = require('algoliasearch');
+
+const client = algoliasearch('YourApplicationID', 'YourAdminAPIKey');
+const index = client.initIndex('your_index_name');
+
+const objects = [{
+  objectID: 1,
+  name: 'Foo'
+}];
+
+index
+  .saveObjects(objects)
+  .then(({ objectIDs }) => {
```

Comment:
Although we use this syntax somewhere in the docs, I think using plain object (e.g. `result`) instead of destructuring is accessible to more people.